### PR TITLE
Feature/add feelings

### LINKED ISSUE #117: Make the "My feelings" page

### PR DESCRIPTION
**Pull request by pulkitvyas08**
Resolves #117

### CODE REVIEW

**Comment by pulkitvyas08** [comment]
WIP, 90% there, TODO: remove feelings, take to dates page

**Comment by tijlleenders** [comment]
Great news @pulkitvyas08 !
Getting the storage wired up is a milestone.

**Review by tijlleenders — APPROVED** [review]

**Comment by tijlleenders** [comment]
@pulkitvyas08 
I've added the remaining work in #275 
Please auto-assign if you're finishing this up or suggest someone else.

File: src/api/FeelingsAPI/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { getJustDate } from '@utils';
+import { db } from '@models/index';
+
+export const resetDatabase = () => db.transaction('rw', db.feelingItems, async () => {
```
[the remark this answers is not in the document]
Comment (pulkitvyas08) [reply src/api/FeelingsAPI/index.js:4]: This is definitely an error, thanks for pointing out

File: src/models/db.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+// @ts-ignore
+import { FeelingItem } from './FeelingItem.ts';
+
+export class FeelingsDB extends Dexie {
+  feelingsCollection!: Table<FeelingItem, number>;
+
+  constructor() {
+    super('FeelingsDB');
+    this.version(1).stores({
+      feelingsCollection: '++id, content, category, date',
```
[the remark this answers is not in the document]
Comment (pulkitvyas08) [reply src/models/db.ts:11]: For the current state of the API `feelingsCollection` is good enough, in the event that we extend it to other things we might take this into consideration

**Pull request merged**